xds/src/main/java/io/grpc/xds/XdsClientWrapperForServerSds.java

```diff
@@ -116,8 +115,6 @@ public void createXdsClientAndStart() throws IOException {
     newServerApi = serverInfo.isUseProtocolV3() && experimentalNewServerApiEnvVar;
     String grpcServerResourceId = bootstrapInfo.getGrpcServerResourceId();
     if (newServerApi && grpcServerResourceId == null) {
-      reportError(
-          Status.INVALID_ARGUMENT.withDescription("missing grpc_server_resource_name_id value"));
       throw new IOException("missing grpc_server_resource_name_id value");
```

Comment:
I know this is preexisting, so you don't need to change now. But this string should probably include "xds bootstrap" in it somewhere.